### PR TITLE
Create CITATION.cff

### PR DESCRIPTION
**Pull request by LuKC1024**

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    orcid: "https://orcid.org/0000-0001-7078-9287"
+  - family-names: "Krishnamurthi"
+    given-names: "Shriram"
+    orcid: "https://orcid.org/0000-0001-5184-1975"
+  doi: "10.22152/programming-journal.org/2022/6/8"
+  journal: "The Art, Science, and Engineering of Programming"
+  title: "Types for Tables: A Language Design Benchmark"
+  issue: 2
+  volume: 6
+  year: 2022
```
Comment (bennn) [inline CITATION.cff:35]: if possible, add the pages from the proceedings: `8:1--8:30`

(page numbers are good way to certify that the material was really published)

Comment (LuKC1024) [reply CITATION.cff:35]: Added. [d7c7d3d](https://github.com/brownplt/B2T2/pull/39/commits/d7c7d3d218d59d86dd3f19d5e4e23a0d93dd0cc7)

Comment (LuKC1024) [reply CITATION.cff:35]: Yes! After looking closer at the [schema](https://github.com/citation-file-format/citation-file-format/blob/52647a247e9b1a5b04154934f39615b5ee8c4d65/schema.json#L1523) that the `"start"` field, which is supposed to store the first page of the paper, can store a string.
```
                "start": {
                    "anyOf": [
                        {
                            "type": "integer"
                        },
                        {
                            "minLength": 1,
                            "type": "string"
                        }
                    ],
                    "description": "The start page of the work."
                },
```

Comment (LuKC1024) [reply CITATION.cff:35]: Fixed in 5b43424

**Review by bennn — APPROVED** [review]

**Pull request merged**